core/src/main/scala/org/ensime/indexer/SearchService.scala

```diff
@@ -30,6 +31,8 @@ class SearchService(
 ) extends ClassfileIndexer
     with ClassfileListener
     with SLF4JLogging {
+
+  private val QUERY_TIMEOUT = 30 seconds
```

Comment:
So what do you suggest? Increase? Block infinitely? We need one for Await.result/Await.ready